package.json

```diff
@@ -46,32 +46,32 @@
     "jest-junit": "^5.1.0",
     "jest-silent-reporter": "^0.1.1",
     "jquery": "^3.2.1",
-    "karma": "^2.0.0",
+    "karma": "^3.0.0",
     "karma-browserify": "^5.1.1",
     "karma-chrome-launcher": "^2.1.1",
     "karma-mocha": "^1.3.0",
     "left-pad": "^1.1.1",
-    "lerna": "2.11.0",
+    "lerna": "^3.3.2",
```

Comment:
In `lerna.json` we have `3.4.0`. I know yarn will lock to certain version, but wanted to note that anyway

Reply:
want me to remove the caret again?

Reply:
nah, just `^3.4.0`, but just confirmed that with lockfile, so disregard that :)